model/model.py

```diff
@@ -1,15 +1,17 @@
 import json
 from typing import List
 from model.events.lamp_event import LampEvent
+from model.scheduler import *
 
 class Model():
-    def __init__(self, data) -> None:
+    def __init__(self, data: str, scheduler: Scheduler) -> None:
         self.user_data = self.read_data(data)
-        self.schedule = self.update_schedule(self.user_data)
+        self.schedule: Schedule = self.update_schedule(self.user_data, scheduler)
 
     def mainloop(self):
+        pass
         #self.user_data = self.update_data()
-        self.schedule = self.update_schedule(self.user_data)
+        # self.schedule = self.update_schedule(self.user_data,)
```

Comment:
do you need this line at all, because it is commented?

Reply:
Not sure if these are needed but not implemented yet